testflinger.conf.example

```diff
@@ -1,8 +1,8 @@
 # These values can be used in place of environment variables to configure
 # Testflinger. If MONGO_URI is not specified, then it will be built using the
 # other values specified
-# MONGODB_USERNAME=testflinger
-# MONGODB_PASSWORD=testflinger
-# MONGODB_DATABASE=testflinger_db
-# MONGODB_HOST=mongo
-# MONGO_URI = "mongodb://mongo:27017/testflinger_db"
+# MONGODB_USERNAME="testflinger"
+# MONGODB_PASSWORD="testflinger"
+# MONGODB_DATABASE="testflinger_db"
+# MONGODB_HOST="mongo"
+# MONGODB_URI = "mongodb://mongo:27017/testflinger_db"
```

Comment:
Just a question about formatting. Why does the last statement have "=" wrapped with the spaces, but the others don't?

Reply:
fixed, thanks!

Reply:
Thanks for fixing this!